pyccel/parser/semantic.py

```diff
@@ -1917,7 +1918,7 @@ def _get_indexed_type(self, base, args, expr):
                 raise errors.report(f"Unknown annotation base {base}\n"+PYCCEL_RESTRICTION_TODO,
                         severity='fatal', symbol=expr)
             rank = 1
-            if len(args) == 2 and args[1] is LiteralEllipsis():
+            if (len(args) == 2 and args[1] is LiteralEllipsis()) or (len(args) > 0):
```

Comment:
This seems to have come from your other branch

Reply:
Fixed : 0bca7ba